internal/rbd/nodeserver.go

```diff
@@ -60,6 +61,22 @@ type stageTransaction struct {
 	devicePath string
 }
 
+var (
+	release = ""
```

Comment:
call this `kernelRelease` or `kernelVersion` please. Just `release` is confusing (was it attached?).